test/e2e/func.js

```diff
@@ -78,9 +78,8 @@ function buildFirefoxWebdriver () {
 
 async function getExtensionIdChrome (driver) {
   await driver.get('chrome://extensions')
+  // const extensionId = await driver.executeScript('return document.querySelector("extensions-manager").shadowRoot.querySelector("cr-view-manager extensions-item-list").shadowRoot.querySelector("extensions-item:nth-child(2)").getAttribute("id")')
```

Comment:
@dennis00010011b let's remove this commented line

Reply:
@vbaranov Done